packages/core/src/telemetry/uiTelemetry.test.ts

```diff
@@ -43,7 +43,7 @@ const createFakeCompletedToolCall = (
       status: 'success',
       request,
       tool,
-      invocation: tool.build({}),
```

Comment:
seems unrelated. revert if so.

Reply:
`tool.build({})` validates parameters now, and the mock tool expects one parameter. However, we can make this configurable. Would you prefer that?